SharpPcap/GetPacketStatus.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+namespace SharpPcap
+{
+    public enum GetPacketStatus
+    {
+        NoDataPresent = 0,
```

Comment:
For offline files the term `NoDataPresent`, would be confused with `PCAP_ERROR_BREAK`

```suggestion
        Timeout= 0,
```

Reply:
Makes sense I think I've improved this a bit, will push shortly.